cypress/integration/accounts/chart_of_accounts.js

```diff
@@ -0,0 +1,139 @@
+context("Chart Of Accounts", () => {
+    before(() => {
+        cy.login();
```

Comment:
@yadavyk @Komal-Saraf0609  this whole duplicate file is re-committed as separate test.  


Always create new branch instead of reusing old branches 😅